charts/operator-wandb/values.yaml

```diff
@@ -1288,6 +1288,175 @@ frontend:
+    GORILLA_ANALYTICS_SINK:
+      value: '{{ include "wandb.mysql" . | trim }}'
+    GORILLA_HISTORY_STORE:
+      value: 'http://{{ .Release.Name }}-parquet:8087/_goRPC_,{{ include "wandb.mysql" . | trim }}'
+    GORILLA_METADATA_CACHE:
+      value: '{{ include "wandb.redis.connectionString" . | trim }}'
+    REDIS:
+      value: '{{ include "wandb.redis.connectionString" . | trim }}'
+    GORILLA_AUDITOR_CACHE:
+      value: '{{ include "wandb.redis.connectionString" . | trim }}'
+    GORILLA_SETTINGS_CACHE:
+      value: '{{ include "wandb.redis.connectionString" . | trim }}'
+    GORILLA_CACHE:
+      value: '{{ include "wandb.redis.connectionString" . | trim }}'
+    GORILLA_FILE_HOST:
+      value: "{{ .Values.global.host }}"
+    OPERATOR_ENABLED:
+      value: "true"
+    LOGGING_ENABLED:
+      value: "true"
+    GORILLA_ONPREM:
+      value: "true"
+    GORILLA_FILE_STORE:
+      value: '{{ (include "wandb.bucket" . | fromYaml).url }}'
+    GORILLA_STORAGE_BUCKET:
+      value: '{{ (include "wandb.bucket" . | fromYaml).url }}'
+    GORILLA_TRACER:
+      value: '{{ include "wandb.otelTracesEndpoint" . | trim }}'
+
+    ### I'm unsure the following are necessary
+    G_HOST_IP:
+      valueFrom:
+        fieldRef:
+          fieldPath: status.hostIP
+    GORILLA_CUSTOMER_SECRET_STORE_K8S_CONFIG_NAMESPACE:
+      valueFrom:
+        fieldRef:
+          fieldPath: metadata.namespace
+    GORILLA_TSHIRT_SIZE:
+      value: "{{ .Values.global.size }}"
+
+    ##### The following are unique/extra critical to metric-observer
+    GORILLA_METRIC_OBSERVER_STORE:
+      value: '{{ include "wandb.redis.connectionString" . | trim }}' 
+    GOMAXPROCS:
+      # this is important in determining how MO threads get spun up in response to 
+      # and thus processing speed
+      value: request.cpu 
+    GORILLA_FRONTEND_HOST:
+      value: "{{ .Values.global.host }}"
+    # buckets are used for overflow of messages
+    BUCKET_ACCESS_KEY:
+      valueFrom:
+        secretKeyRef:
+          name: '{{ (include "wandb.bucket" . | fromYaml).secretName }}'
+          key: '{{ (include "wandb.bucket" . | fromYaml).accessKeyName }}'
+          optional: true
+    BUCKET_SECRET_KEY:
+      valueFrom:
+        secretKeyRef:
+          name: '{{ (include "wandb.bucket" . | fromYaml).secretName }}'
+          key: '{{ (include "wandb.bucket" . | fromYaml).secretKeyName }}'
+          optional: true
+    
+    # I'm pretty sure it's safe to skip these - we use these during
+    # setup of the slack integration, but by the time MO runs the slack integration,
+    # we already have all the info we need embedded in the integration
+    # GORILLA_SLACK_CLIENT_ID:
+    #    value: "{{ required "slack client id required" .Values.secretRefs.slack.alertsApp.clientId }}"
+    # GORILLA_SLACK_SECRET:
+    #    value: "{{ required "slack secret required" .Values.secretRefs.slack.alertsApp.secret }}"
+
+    # TODO - unclear if/how AZURE_STORAGE_KEY one is used
```

Comment:
re-reading this now, I suspect this is needed for buckets, but looking at the code earlier, it wasn't obvious that it's tied to normal bucket usage

Reply:
k, since we need buckets I'll plan to add that back in